src/BizHawk.Client.EmuHawk/MainForm.Designer.cs

```diff
@@ -2758,7 +2766,8 @@ private void InitializeComponent()
 		private BizHawk.WinForms.Controls.ToolStripMenuItemEx AmstradCPCNonSyncSettingsToolStripMenuItem;
 		private BizHawk.WinForms.Controls.ToolStripSeparatorEx toolStripSeparator8;
 		private System.Windows.Forms.ToolStripMenuItem CaptureLuaMenuItem;
-		private System.Windows.Forms.ToolStripMenuItem AutosaveLastSlotMenuItem;
 		private ToolStripSeparatorEx toolStripSeparator24;
+		private System.Windows.Forms.ToolStripMenuItem DownloadCoresMenuItem;

```

Comment:
Revert unrelated changes and/or insert the menu item in `MainForm_Load`.

Reply:
Dealing with the designer is giving me mild forms of permanent damage, I'll just quote from the file itself `do not modify the contents of this method with the code editor`. See also the disclaimer in the PR description.